src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Nav from './site/Nav';
+import { Route, BrowserRouter } from "react-router-dom";
+import AdminLogin from './admin/loginPage/AdminLogin';
+import AdminPage from './admin/adminPage/AdminPage';
+import "../css/app.css";
+
+const App = () => {
+  return (
+    <BrowserRouter>
+      <div>
+        <Nav />
+        <Route exact path='/'>
+          <h1>React Page </h1>
+        </Route>
```

Comment:
this is great work. just on small thing you can change is this route **/** as it going to conflict the one I did so I think it is better to remove it

Reply:
will not fail to run when I remove it @maximeish? how about we remain as it is and by the time of merging into develop we remove conflicts manually?